source/jsonnet/northern-ireland/lib/common_rules.libsonnet

```diff
@@ -1,4 +1,14 @@
 {
+  lastBirthdayAgeOver(age): {
```

Comment:
over implies it is exclusive of the age, but the condition is inclusive

Reply:
Agreed but this was discussed a while back, and we excepted that `over` would be fine for age.

Reply:
https://github.com/ONSdigital/eq-questionnaire-schemas/pull/73#discussion_r426519245